### PR TITLE
COMMERCE-9093 Discontinued products' info should be shown also on the product card

### PR DESCRIPTION
**Pull request by alessiorendina**
https://issues.liferay.com/browse/COMMERCE-9093

### CODE REVIEW

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp [hunk 1]
```diff
@@ -127,6 +127,14 @@ CPCatalogEntry cpCatalogEntry = cpContentHelper.getCPCatalogEntry(request);
 						</div>
 					</div>
 
+					<div class="autofit-col">
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp:130]: is this the right position in the end?

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp]: Talked with design team, and now the position is good.

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:close

**Pull request closed without merging**